modules/caddyhttp/metrics.go

```diff
@@ -160,6 +159,30 @@ func sanitizeCode(code int) string {
 	return strconv.Itoa(code)
 }
 
+// Only support the list of "regular" HTTP methods, see
+// https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
+var methodMap = map[string]string{
```

Comment:
Can we possibly not repeat this in the other metrics.go file as well? I guess that might involve exporting something but maybe that's OK... (kind of a bummer that the core caddy package uses it...)

Reply:
@mholt that's why I asked about your feelings on `internal/` in the description 😉

Reply:
Ahhh wow totally missed that, sorry. I'll circle back to this shortly!

Reply:
@hairyhenderson I'm curious what the implementation using `internal/` would look like. I think we should definitely be careful/limit what we put in there, but I think the method map is definitely one of those obscure things that doesn't make sense to truly export for other modules to use. :+1:

Reply:
@mholt ok, I'll hack something up and update this PR